packages/admin/dashboard/src/i18n/translations/fr.json

```
@@ -0,0 +1,2748 @@
+{
+	"$schema": "./$schema.json",
+	"general": {
+		"ascending": "Ascendant",
+		"descending": "Descendant",
+		"add": "Ajouter",
+		"start": "Début",
+		"end": "Fin",
+		"open": "Ouvrir",
+		"close": "Fermer",
+		"apply": "Appliquer",
+		"range": "Plage",
+		"search": "Rechercher",
+		"of": "de",
+		"results": "résultats",
+		"pages": "pages",
+		"next": "Suivant",
+		"prev": "Précédent",
+		"is": "est",
+		"timeline": "Timeline",
+		"success": "Succès",
+		"warning": "Attention",
+		"tip": "Astuce",
+		"error": "Erreur",
+		"select": "Sélectionner",
+		"selected": "Sélectionné",
+		"enabled": "Activé",
+		"disabled": "Désactivé",
+		"expired": "Expiré",
+		"active": "Actif",
+		"revoked": "Revoqué",
+		"new": "Nouveau",
+		"modified": "Modifié",
+		"added": "Ajouté",
+		"removed": "Removed",
+		"admin": "Admin",
+		"store": "Boutique",
+		"details": "Détails",
+		"items_one": "{{count}} item",
+		"items_other": "{{count}} items",
+		"countSelected": "{{count}} sélectionné",
+		"countOfTotalSelected": "{{count}} de {{total}} sélectionné",
+		"plusCount": "+ {{count}}",
+		"plusCountMore": "+ {{count}} plus",
+		"areYouSure": "Êtes-vous sûr ?",
+		"noRecordsFound": "Aucun enregistrement trouvé",
+		"typeToConfirm": "Veuillez entrer {{val}} pour confirmer :",
+		"noResultsTitle": "Aucun résultat",
+		"noResultsMessage": "Essayez de changer les filtres ou la requête de recherche",
+		"noSearchResults": "Aucun résultat de recherche",
+		"noSearchResultsFor": "Aucun résultat de recherche pour <0>'{{query}}'</0>",
+		"noRecordsTitle": "Aucun enregistrement",
+		"noRecordsMessage": "Il n'y a pas d'enregistrements à afficher",
```

```suggestion
		"noRecordsMessage": "Il n'y a pas d'enregistrement à afficher",
```
for consistency with previous singular usage